cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/timeseries/FactTable.java

```diff
@@ -310,6 +346,18 @@ private FuzzyRowFilter createFuzzyRowFilter(FactScan scan, byte[] startRow, bool
     return new FuzzyRowFilter(ImmutableList.of(new ImmutablePair<byte[], byte[]>(startRow, fuzzyRowMask)));
   }
 
+
+  @Nullable
+  private FuzzyRowFilter createFuzzyRowFilter(FactScan scan, byte[] startRow, boolean anyAggGroup,
+                                              boolean fuzzyMeasureName) {
+    // we need to always use a fuzzy row filter as it is the only one to do the matching of values
+
+    byte[] fuzzyRowMask = codec.createFuzzyRowMask(scan.getTagValues(), fuzzyMeasureName ? null : scan.getMeasureName(),
```

Comment:
do we need extra param fuzzyMeasureName? I think we assume it is fuzzy if scan.getMeasureName() is null